sdk/storage/azure-storage-common/inc/azure/storage/common/crypt.hpp

```diff
@@ -7,32 +7,21 @@
 #include <string>
 #include <vector>
 
+#include "azure/core/cryptography/hash.hpp"
 #include <azure/core/base64.hpp>
 
 namespace Azure { namespace Storage {
 
-  class Crc64 {
+  class Crc64Hash : public Azure::Core::Cryptography::Hash {
   public:
-    void Update(const uint8_t* data, std::size_t length);
-    void Concatenate(const Crc64& other);
-
-    std::vector<uint8_t> Digest() const;
-
-    static std::vector<uint8_t> Hash(const uint8_t* data, std::size_t length)
-    {
-      Crc64 instance;
-      instance.Update(data, length);
-      return instance.Digest();
-    }
-
-    static std::vector<uint8_t> Hash(const std::string& data)
-    {
-      return Hash(reinterpret_cast<const uint8_t*>(data.data()), data.length());
-    }
```

Comment:
Can we keep this?

Reply:
Will folks typically try to compute Crc64 hash for text string data? Or is it similar to other hashing (like md5) which is meant for files/blobs that tend to binary/bytes?

These are convenience methods which we can easily add later, and so can the end-customer if they really want to.

It's not too bad atm:
```C++
CrcHash instance;
auto hash = instance.Final(data, length);
```
versus
```C++
auto hash = CrcHash::Hash(data, length);
```